ynr/apps/official_documents/forms.py

```diff
@@ -10,4 +11,8 @@ class Meta:
 
         widgets = {"ballot": forms.HiddenInput()}
 
+    uploaded_file = forms.FileField(
+        validators=[FileExtensionValidator(allowed_extensions=["pdf"])]
```

Comment:
If we can convert docx, html, etc, why not allow users to upload them?

Reply:
I think this gets back to the idea that the upload form is the 'less common' way - but happy to discuss. I don't really see a need for this if we have the functionality to convert.